core/embed/rust/src/ui/model_mercury/flow/confirm_output.rs

```diff
@@ -41,6 +44,47 @@ impl FlowState for ConfirmOutput {
         *self as usize
     }
 
+    fn handle_swipe(&'static self, direction: SwipeDirection) -> StateChange {
+        match (self, direction) {
+            (Self::Address, SwipeDirection::Left) => Self::Menu.swipe(direction),
+            (Self::Address, SwipeDirection::Up) => self.return_msg(FlowMsg::Confirmed),
+            (Self::Menu, SwipeDirection::Right) => Self::Address.swipe(direction),
+            (Self::Menu, SwipeDirection::Left) => Self::AccountInfo.swipe(direction),
```

Comment:
Not sure we want to go from `Menu` to one of the items by swiping left.

Reply:
https://github.com/trezor/trezor-firmware/pull/4095/commits/d2cded804019bbb5fe3d646216db582943bd5826#diff-5cb317ed79ea16b58238d2743af510c9f82a27c2764811d83f532f8d8358b6f6L95